pkg/base/logger/runtime_formatter.go

```diff
@@ -0,0 +1,61 @@
+package logger
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// FunctionKey holds the function field
+const FunctionKey = "function"
+
+// FileKey holds the file field
+const FileKey = "file"
+
+const (
+	logrusStackJump          = 4
+	logrusFieldlessStackJump = 6
+)
+
+// RuntimeFormatter decorates log entries with function name and package name (optional) and line number (optional)
```

Comment:
The package name should be provided as [fields](https://pkg.go.dev/github.com/sirupsen/logrus#readme-fields) 
Because of `json` log formatter.

Reply:
Can you please write the comment in details

Reply:
The services like kibana parse json logs in the elastic index, to have package name parsed as expected it should be field (json key\value) otherwise it's hard to parse package name from plain string.

Reply:
And what't wrong with this package name?

Reply:
I was thinking it's for log entry but it for package function.